scripts/deprecations-checker.sh

```diff
@@ -22,7 +22,7 @@
 # Here, it is used to check if a deprecated function, variable, constant or field is used.
 
 # Run staticcheck 
-set -e
+set -ex
```

Comment:
Do we need expanded output in this script?

Reply:
good call, we don't. i had it on for debugging